<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Dialog{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.203704 rgba(36, 57, 187, 255), stop:1 rgba(162, 164, 255, 255));}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>791</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 100 16pt &quot;Space Grotesk SemiBold&quot;;color:rgb(253, 228, 255); white-space: normal; </string>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Space Grotesk SemiBold'; font-size:16pt; font-weight:96; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:12px; margin-right:12px; -qt-block-indent:0; text-indent:0px;&quot;&gt; Classification results.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0, 0);
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: rgb(253, 228, 255);
font: 12pt &quot;Space Grotesk&quot;;
color: rgb(253, 228, 255);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="arrowType">
    <enum>Qt::LeftArrow</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="lenguageComboBox">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(36, 57, 187);
font: 10pt &quot;Space Grotesk&quot;;
color:rgb(244, 216, 255);
border-style: outset;
border-width: 0px;
border-radius: 10px;
border-color: rgb(253, 228, 255);



</string>
   </property>
   <item>
    <property name="text">
     <string>English</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Espa√±ol</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>851</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
    color: rgb(13, 0, 77);
    font: 12px;
</string>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="continueButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>740</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-width: 0px;
border-radius:10px;
border-color: rgb(251, 234, 255);
background-color: rgb(251, 234, 255);
font: 11pt &quot;Space Grotesk&quot;;
color: rgb(0, 0, 127)
</string>
   </property>
   <property name="text">
    <string>Continue classifying</string>
   </property>
  </widget>
  <widget class="QPushButton" name="emptyButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>740</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-width: 0px;
border-radius:10px;
border-color: rgb(251, 234, 255);
background-color: rgb(34, 57, 187);
font: 11pt &quot;Space Grotesk&quot;;
color: rgb(251, 234, 255);
text-align: center;
</string>
   </property>
   <property name="text">
    <string>Empty table</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>815</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

image: url(:/foto/logo2.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="C:/Users/lucia/Desktop/test.qrc">:/foto/lenguahe2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="C:/Users/lucia/Desktop/test.qrc"/>
 </resources>
 <connections/>
</ui>
